playground.md

```diff
@@ -1,18 +1,21 @@
 # Playground
-Playground provides a standalone MARO running environment, it also includes examples of some predefined scenarios and quick-start tutorial.
 
-## Main service in playground
+The playground provides a standalone MARO running environment, it also includes examples of some predefined scenarios and quick-start tutorials.
```

Comment:
The playground provides a standalone MARO environment along with several predefined scenarios and quick-start tutorials.

Reply:
how about `predefined scenario examples and ...` ?

Reply:
done